src/rtf-plugins/fixture-managers/yarpmanager/YarpFixManager.cpp

```diff
@@ -61,7 +61,7 @@ bool YarpFixManager::setup(int argc, char** argv) {
         enableRestrictedMode();
 
         // load the fixture (application xml)
-        char szAppName[255];
+        char* szAppName = new char [10];
```

Comment:
`255` was a random number, you can't know how much long the name is.
It is only a placeholder in fact the real allocation is made in `KnowledgeBase::addApplication` where we can know exactly how much long is the name.


Probably befor